docs/decisions/0003-static-builds-to-client-side-react.rst

```diff
@@ -0,0 +1,55 @@
+============================================================
+3. Move from static builds (Gatsby) to client-side React app
+============================================================
+
+******
+Status
+******
+
+Accepted
+
+*******
+Context
+*******
+
+Currently, the Enterprise and Programs learner portals are created using Gatsby, a static site generator based on React. Gatsby works by sourcing data (e.g., from external APIs) and creates static web pages (i.e., HTML, CSS, JavaScript) during the build process. This contrasts from a standard React application (or "microfrontend") where data is fetched client-side, or after the app has already been loaded in the browser. The Enterprise and Programs learner portals both utilize a custom Gatsby plugin to fetch the necessary data about learner portal configurations from the ``portal-designer`` service (i.e., ``gatsby-source-portal-designer``).
```

Comment:
Same comment on present vs. past tense for this doc. I'm fine with whatever we decide if we're consistent 🤷‍♂

Reply:
I will update this to reflect and link to the decision outlined in ADR 2 about no longer using portal-designer.